tardis/plasma/tests/conftest.py

```diff
@@ -213,12 +210,10 @@ def electron_densities(phi_saha_lte, partition_function, number_density):
 
 @pytest.fixture
 def delta(w, ionization_data, beta_rad, t_electrons, t_rad, beta_electron,
-          levels, chi_0):
-    delta_input = None
-    delta_module = RadiationFieldCorrection(None)
-    delta_module.chi_0_species = (2, 2)
+          levels):
+    delta_module = RadiationFieldCorrection(chi_0_species=(2,2))
     return delta_module.calculate(w, ionization_data, beta_rad, t_electrons,
-        t_rad, beta_electron, delta_input, chi_0)
+        t_rad, beta_electron)
 
```

Comment:
@wkerzendorf I'm not sure I really understand how default arguments work with classes. When I do this with the `delta` module for the tests (because the testing atomic data file only has He data), the test fails because in `RadiationFieldCorrection.calculate()`, the value `chi_0_species` is still interpreted as the default ((20,2)). What's the correct way to do this?